hetznerbot/helper/hetzner.py

```diff
@@ -181,6 +181,16 @@ def check_offer_for_subscriber(session, subscriber):
                 )
             )
         )
+        .filter(
+            Offer.cpu.in_(
+                session.query(Cpu.name)
+                .filter(Cpu.cores >= subscriber.cores)
+                .filter(Cpu.threads >= subscriber.threads)
+                .filter(Cpu.release >= subscriber.release)
+                .filter(Cpu.multi_thread_rating >= subscriber.multi_rating)
+                .filter(Cpu.single_thread_rating >= subscriber.single_rating)
```

Comment:
We might also change this query in a way, that the offer will be considered if the CPU is unknown.

Without this, we won't ever notice that there're new CPUs, since they won't show up in the bot

Reply:
yeah I was thinking about maybe adding an admin notification job that shows a new unknown cpu. then another command to set the stats after checking them

Reply:
I agree, the admin notification job would be the cleanest solution :)